ci/pipeline.yml

```diff
@@ -74,31 +83,32 @@ jobs:
     put: slack
     params:
       text: |
-        :white_check_mark: Successfully deployed development BOSH
+        :white_check_mark: Successfully deployed Master BOSH
         <$ATC_EXTERNAL_URL/teams/$BUILD_TEAM_NAME/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME|View build details>
       channel: ((slack-channel))
       username: ((slack-username))
       icon_url: ((slack-icon-url))
 
-- name: common-releases-development
+- name: common-releases-master
   serial: true
   plan:
   - in_parallel:
     - get: certificate
       resource: ca-cert-store
     - get: bosh-deployment
+      passed:
+      - deploy-master-bosh
       trigger: true
     - get: bosh-config
-      resource: bosh-config-development
     - get: terraform-yaml
-      resource: terraform-yaml-development
+      resource: terraform-yaml-production
```

Comment:
`common-releases-master` doesn't currently `get` these resources - is this just for resource passing, or ...?

Reply:
It's just for ordered resource passing.

Reply:
should they `trigger` then?

Reply:
@bengerman13 can you specify which ones you want me to add the triggers on?

Reply:
all the same ones the upstream and downstream jobs trigger on. I think the ones you're missing are bosh-config, bosh-deployment, and terraform-yaml

Reply:
done!